hedyorg/hedy
# 🪲 Improve unquoted error text

Fixes #3444 
This PR changes the text of the unquoted error to display only the segments which are not quoted correctly. Also, it removes the incorrectly doubled backslashes.

**How to test**
In level 6 run the following command:
```
print 'This is ' unquoted ' and '
```
The error message should only mention the word `unquoted` and not the parts which are correctly quoted.

Again, in level 6 execute the following command:
```
print '5 divided by 5 is ' 5 \ 5
```
Ensure that the error message points only to the `5 \ 5` part and that the message contains a single backslash (exactly as it appears in the code editor).
